src/Functions/FunctionsJSON.cpp

```diff
@@ -803,25 +821,28 @@ struct JSONExtractTree
         }
     };
 
-    class LowCardinalityNode : public Node
+    class LowCardinalityFixedStringNode : public Node
     {
     public:
-        LowCardinalityNode(DataTypePtr dictionary_type_, std::unique_ptr<Node> impl_)
-            : dictionary_type(dictionary_type_), impl(std::move(impl_)) {}
+        LowCardinalityFixedStringNode(const size_t fixed_length_) : fixed_length(fixed_length_) {}
```

Comment:
Looks like the last warning in CI is in tidy clang-15 debug is:
```
Sep 27 18:02:45 /build/build_docker/../src/Functions/FunctionsJSON.cpp:827:9: error: single-argument constructors must be marked explicit to avoid unintentional implicit conversions [google-explicit-constructor,-warnings-as-errors]
Sep 27 18:02:45         LowCardinalityFixedStringNode(const size_t fixed_length_) : fixed_length(fixed_length_) {}
Sep 27 18:02:45         ^
Sep 27 18:02:45         explicit
```

```suggestion
        explicit LowCardinalityFixedStringNode(const size_t fixed_length_) : fixed_length(fixed_length_) {}
```